src/Lykke.Service.TradeVolumes/Controllers/TradeVolumesController.cs

```diff
@@ -94,8 +95,10 @@ public TradeVolumesController(ITradeVolumesCalculator tradeVolumesCalculator)
 
             if (fromDate.Kind == DateTimeKind.Unspecified)
                 fromDate = DateTime.SpecifyKind(fromDate, DateTimeKind.Utc);
+            fromDate = fromDate.RoundToHour();
             if (toDate.Kind == DateTimeKind.Unspecified)
```

Comment:
From my experience toDate.Kind can be  DateTimeKind.Local or Utc, but not Unspecified